doc/merging.rst

```diff
@@ -0,0 +1,56 @@
+How to merge a branch after approving the PR
```

Comment:
This implies that *you* have approved the PR. Change to 'How to merge approved PRs'.